aws/elb_test.go

```diff
@@ -54,14 +54,14 @@ func TestListELBs(t *testing.T) {
 
 	elbNames, err := getAllElbInstances(session, region, time.Now().Add(1*time.Hour*-1))
 	if err != nil {
-		assert.Fail(t, "Unable to fetch list of Auto Scaling Groups")
+		assert.Fail(t, "Unable to fetch list of ELBs")
```

Comment:
In all of these `assert.Fail`, you should include the error so we can tell what went wrong.

```go
assert.Fail(t, "Unable to fetch list of ELBs: %v", err)
```